db/institutions.yml

```diff
@@ -262,4 +262,13 @@ nyu:
   name: New York University
   state: NY
   zip: 10003
+qmul:
+  aka: [QMUL, Queen Mary University of London, Queen Mary College, QMC]
+  city: London
+  country: UK
+  schools: {physastro: The School of Physics and Astronomy}
```

Comment:
schools and departments are now dicts of dicts.  please use other schools as an example